src/main/java/appeng/fluids/helper/IAEFluidInventory.java

```diff
@@ -0,0 +1,11 @@
+
+package appeng.fluids.helper;
+
+
+import net.minecraftforge.fluids.capability.IFluidHandler;
+
+
+public interface IAEFluidInventory
+{
+	public void onFluidInventoryChanged( IFluidHandler inventory );
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Redundant 'public' modifier. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.modifier.RedundantModifierCheck)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Avoid modifiers which are implied by the context [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=pmd%3AUnusedModifier)